dashboard/config/routes.rb

```diff
@@ -6,6 +6,8 @@ module OPS
 end
 
 Dashboard::Application.routes.draw do
+  resources :survey_results, only: [:create], defaults: { format: 'json' }
```

Comment:
We mostly don't handle multiple formats via `respond_to`... for actions like this that expect to only see json, we just ignore the request format type and always render json (see ApiController for examples).